pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -42,31 +36,10 @@ func (h *Helper) InstallFlags(flags *pflag.FlagSet) {
 // from the local dockercfg file
 func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfiguration, bool) {
 	glog.V(3).Infof("Locating docker auth for image %s and type %s", imageName, authType)
-	var dockercfgPath string
-	if pathForAuthType := os.Getenv(authType); len(pathForAuthType) > 0 {
-		dockercfgPath = GetDockercfgFile(pathForAuthType)
-	} else {
-		dockercfgPath = GetDockercfgFile("")
-	}
-	if len(dockercfgPath) == 0 {
-		glog.V(3).Infof("Could not locate a docker config file")
-		return docker.AuthConfiguration{}, false
-	}
-	if _, err := os.Stat(dockercfgPath); err != nil {
-		glog.V(3).Infof("Problem accessing %s: %v", dockercfgPath, err)
-		return docker.AuthConfiguration{}, false
-	}
-
-	var cfg credentialprovider.DockerConfig
-	var err error
-	if strings.HasSuffix(dockercfgPath, kapi.DockerConfigJsonKey) || strings.HasSuffix(dockercfgPath, "config.json") {
-		cfg, err = readDockerConfigJson(dockercfgPath)
-	} else if strings.HasSuffix(dockercfgPath, kapi.DockerConfigKey) {
-		cfg, err = readDockercfg(dockercfgPath)
-	}
 
+	cfg, err := credentialprovider.ReadDockerConfigFile()
```

Comment:
you've deleted the logic that determines what path we actually look for the credential file in, based on the value of the authType env variable.